lib/v2/screens/sign_up/viewmodels/signup_bloc.dart

```diff
@@ -77,4 +88,19 @@ class SignupBloc extends Bloc<SignupEvent, SignupState> {
       }
     }
   }
+
+  Stream<SignupState> createAccount(SignupState currentState, String? phoneNumber) async* {
+    final currentAddPhoneNumberState = currentState.addPhoneNumberState;
+
+    yield currentState.copyWith(addPhoneNumberState: AddPhoneNumberState.loading(currentAddPhoneNumberState));
+
+    final inviteSecret = state.claimInviteState.inviteModel!.inviteSecret;
+    final displayName = state.displayNameState.displayName;
+    final username = state.createUsernameState.username;
```

Comment:
Something tells me that if the user has reached this point, none of those values ​​should be null or can they?

Reply:
Phone number is optional and may be null (I guess) - the rest not null

Reply:
That's correct, at this point nothing can be null but the phone number since it's optional.